automated install/basic-install.sh

```diff
@@ -285,10 +330,10 @@ installConfigs() {
 	# Install the configs from /etc/.pihole to their various locations
 	$SUDO echo ":::"
 	$SUDO echo -n "::: Installing configs..."
-	$SUDO mv /etc/dnsmasq.conf /etc/dnsmasq.conf.orig
+	versionCheckDNSmasq
 	$SUDO mv /etc/lighttpd/lighttpd.conf /etc/lighttpd/lighttpd.conf.orig
+	$SUDO cp /etc/.pihole/advanced/lighttpd.conf /etc/lighttpd/lighttpd.conf 
 	$SUDO cp /etc/.pihole/advanced/dnsmasq.conf /etc/dnsmasq.conf
```

Comment:
This line causes a conflict with dnsmasq. We end up with a config file in `/etc/dnsmasq.d/01-pihole.conf` and `/etc/dnsmasq.conf`. `dnsmasq` doesn't like having keywords repeated, so it throws an error.